Frontend/ui-customer/package.json

```diff
@@ -48,12 +48,12 @@
     "husky": "^8.0.1",
     "node-notifier": "^10.0.1",
     "npm-run-all": "^4.1.5",
+    "purty": "^7.0.0",
     "semantic-release": "^19.0.5",
     "webpack": "^4.41.5",
     "webpack-cli": "^3.3.10",
     "webpack-dev-server": "^3.10.1",
-    "webpack-merge": "^5.8.0",
-    "purty": "^7.0.0"
+    "webpack-merge": "^5.8.0"
```

Comment:
why we need this ?

Reply:
it is already present only. just reordered. changed now